README.md

```diff
@@ -22,5 +22,34 @@ grocy:
 7. Restart Home Assistant
 8. Look for the new Grocy sensor in States and use its info
 
+
+## Additional installation instructions for Hass.io users
+
+The configuration is slightly different for users that use Hass.io and the [official Grocy addon](https://github.com/hassio-addons/addon-grocy) from the official Hass.io Add-on store.
+
+1. If you haven't already done so, install Grocy from the Add-on store
+2. In the 'Network' section of the add-on config, input 9192 in the host field [screenshot](https://github.com/kdw2060/grocy/raw/develop/grocy-addon-config.png). Save your changes and restart the add-on.
```

Comment:
I guess this should point to this repo instead

Reply:
done, i supposed the path should point to the master branch so used that